src/phases/query/CollScanSimplifiablePredicate.yml

```diff
@@ -0,0 +1,360 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of collection scan queries with complex predicates
+  that can be simplified by the optimizer.
+
+GlobalDefaults:
+  Database: &Database {^Parameter: {Name: Database, Default: unused}}
+  DocumentCount: &DocumentCount {^Parameter: {Name: DocumentCount, Default: -1}}
+  Repeat: &Repeat {^Parameter: {Name: Repeat, Default: -1}}
+  Collection: &Collection Collection0
+  MaxPhases: &MaxPhases 20
+
+ActorTemplates:
+- TemplateName: FindQueryTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "unused"}}
+    Type: CrudActor
+    Database: *Database
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "ActivePhase", Default: -1}}]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: *Repeat
+          Collection: *Collection
+          Operations:
+          - OperationName: find
+            OperationCommand:
+              Filter: {^Parameter: {Name: "Filter", Default: {invalid: "invalid"}}}
+
+- TemplateName: SimplifiableConjunction
+  Config:
+    ActorFromTemplate:
+      TemplateName: FindQueryTemplate
+      TemplateParameters:
+        Name: {^Parameter: {Name: "Name", Default: "unused"}}
+        ActivePhase: {^Parameter: {Name: "ActivePhase", Default: -1}}
+        Filter:
+          {$and:
+            {^Cycle: {
```

Comment:
Done.

> just because the ^Cycle/^Array notation is not particularly legible

It's an idiomatic genny for loop 🙂

Reply:
> just because the ^Cycle/^Array notation is not particularly legible

Oh I know (and your usage is fine), but folks on Query are not all super comfortable with Genny syntax, and I'm in favor of reducing the mental overhead for someone investigating some other complicated perf issue.